crates/uv-workspace/src/pyproject.rs

```diff
@@ -346,7 +350,8 @@ impl<'de> serde::de::Deserialize<'de> for ToolUvSources {
     }
 }
 
-#[derive(Serialize, Deserialize, OptionsMetadata, Default, Debug, Clone, PartialEq, Eq)]
+#[derive(Deserialize, OptionsMetadata, Default, Debug, Clone, PartialEq, Eq)]
+#[cfg_attr(test, derive(Serialize))]
```

Comment:
Sigh, these need `Serialize` for testing. Wonder if we should just use the debug repr though.

Reply:
`Debug` repr for tests seems fine to me.

Reply:
It's kind of bad because we have glob patterns and the debug repr is very verbose.